test/index.test.js

```diff
@@ -1117,4 +1117,225 @@ describe('index', () => {
             });
         });
     });
+
+    describe('_addWebhook', () => {
+        const oauthToken = 'oauthToken';
+        const scmUri = 'bitbucket.org:arnoldWeber/wyattLib:mybranch';
```

Comment:
So here's where we're coming from with this: there's no difference in how the module operates.

The module doesn't care what format the slug is. Nothing in the code modifies it or checks to see whether there are curly braces or not. It knows that it's going to always be `owner/slug`. It is known that the slug is the UUID. And so the module takes that information as-is and uses it in the request.

So, based on how the interface defines the contract with the user, these test asserts that "given this 'owner/slug' combination, make a request with it".